### PR TITLE
Update cubical

### PR DESCRIPTION
**Pull request by KshitijT**
Commit summary:

1. Updated Cubical base dockerfile to pull from the Cubical version tagged as 1.4.0.
2. Updated the Cubical cab dockerfile according to point 1.
3. Updated the Cubical parameter file consistent with point 1, added parameters out-dir and out-backup.

### CODE REVIEW

**Comment by bennahugo** [comment]
@KshitijT Have you pushed your new image?

**Comment by KshitijT** [comment]
@bennahugo , yup, I have done so now. Apologies for the mixup.

**Comment by bennahugo** [comment]
retest this please

**Comment by SpheMakh** [comment]
@KshitijT did you see my comments?

File: stimela/cargo/cab/cubical/parameters.json [hunk 1]
```diff
@@ -423,6 +423,18 @@
             "dtype": "bool",
             "default": true
         },
+        {
+            "name": "out-backup",
+            "info": "Allow automatic backup of existing output directories. Automatic backup is only used when OUTDIR is used (i.e. OUTNAME doesn't contain any slashes),and it ends with .cc-out (implicitly or explicitly). In this case, existing output directories are renamed to .cc.out.0, .1, etc. ",
+            "dtype": "bool",
+            "default": true
```
[the remark this answers is not in the document]
Comment (KshitijT) [reply stimela/cargo/cab/cubical/parameters.json]: Ok, let me remove it.

File: stimela/cargo/cab/cubical/parameters.json [hunk 2]
```diff
@@ -423,6 +423,18 @@
...
+        },
+        {
+            "name": "out-dir",
```
[the remark this answers is not in the document]
Comment (KshitijT) [reply stimela/cargo/cab/cubical/parameters.json]: Okiedokes. Willfix.

File: stimela/cargo/cab/cubical/parameters.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (KshitijT) [reply stimela/cargo/cab/cubical/parameters.json]: Done.

File: stimela/cargo/cab/cubical/parameters.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (KshitijT) [reply stimela/cargo/cab/cubical/parameters.json]: Done.

**Pull request merged**